index.html

```diff
@@ -1209,6 +1209,29 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            The manifest is fetched and processed on every page load. When the
+            processing is successful, user agents MAY apply updated manifest to
+            any current and future <a>application contexts</a>
+            associated with the application. 
+          </p>
+          <p>
+            When the user agent fetches a manifest as part of an update check,
+            it MUST [=url/equal|compare=] the fetched processed manifest's
+            [=manifest/identity=] with the identity of the app being updated. If
+            it is not equal, the user agent MUST NOT update the installed
+            application's manifest. 
+          </p>
+          <section>
+            <h3>
+              Privacy and security consideration
+            </h3>
+            <p>
+              User agents SHOULD be careful about performing update to manifest
+              fields like app name and icons. Users should be fully aware if
+              such fields are to be be updated to avoid web applications being
+              installed and silently changed to appear as a different one.
+            </p>
```

Comment:
Missing `</section>` tag, which is causing later items to get different section numbers in the table of contents.